py/kubeflow/testing/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- bobgy
+- jlewi
```

Comment:
Per my comment I think it might be better to move in the direction of better separation. With that in mind it might be better not to add additional OWNERs here. At the very least might be better to do this in a follow on PR to avoid blocking this PR.

Reply:
Make sense, will do this in a follow-up PR